pages/references.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Project References</h1>
+    <p class="mb-3">
+      <span class="font-medium">Database Source:</span> Julian Goodare, Lauren
+      Martin, Joyce Miller and Louise Yeoman,
+      <a
+        class="underline hover:text-gray-500"
+        href="http://www.shca.ed.ac.uk/witches/"
+        target="_blank"
+      >
+        'The Survey of Scottish Witchcraft' </a
+      >(archived January 2003)
+    </p>
+    <br />
+    <h2 class="mb-3">Place-Name References</h2>
+    <p class="mb-3">
+      Alexander, W.M.C. & Third Spalding Club (1952). The place-names of
```

Comment:
I think this ampersand needs replaced with `&amp;` - Check the encoding to make sure I've got this write. Making this suggestion from memory

Reply:
done

Reply:
thank you!